src/webview_provider/newLineagePanel.ts

```diff
@@ -84,6 +86,19 @@ export class NewLineagePanel implements LineagePanelView {
     private terminal: DBTTerminal,
   ) {}
 
+  private getValueFromCache(key: string): ModelNode | undefined {
+    if (this.currSession !== env.sessionId) {
```

Comment:
since the `cache` is private, the data in cache will be lost once we reload/change workspace. do we need to handle it based on sessionid?